docs/conceptual/gpu-arch/mi200-performance-counters.md

```diff
@@ -1,455 +1,591 @@
+| `SQ_ACTIVE_INST_LDS` | Qcycles | Number of quad-cycles spent by the SQ instruction arbiter to work on an LDS instruction |
+| `SQ_ACTIVE_INST_VALU` | Qcycles | Number of quad-cycles spent by the SQ instruction arbiter to work on a VALU instruction |
+| `SQ_ACTIVE_INST_SCA` | Qcycles | Number of quad-cycles spent by the SQ instruction arbiter to work on a SALU or SMEM instruction |
+| `SQ_ACTIVE_INST_EXP_GDS` | Qcycles | Number of quad-cycles spent by the SQ instruction arbiter to work on an EXPORT or GDS instruction |
+| `SQ_ACTIVE_INST_MISC` | Qcycles | Number of quad-cycles spent by the SQ instruction aribter to work on a BRANCH or `SENDMSG` instruction |
+| `SQ_ACTIVE_INST_FLAT` | Qcycles | Number of quad-cycles spent by the SQ instruction arbiter to work on a FLAT instruction |
+| `SQ_INST_CYCLES_VMEM_WR` | Qcycles | Number of quad-cycles  spent to send addr and cmd data for VMEM Write instructions |
+| `SQ_INST_CYCLES_VMEM_RD` | Qcycles | Number of quad-cycles  spent to send addr and cmd data for VMEM Read instructions |
+| `SQ_INST_CYCLES_SMEM` | Qcycles | Number of quad-cycles  spent to execute scalar memory reads |
+| `SQ_INST_CYCLES_SALU` | Qcycles  | Number of quad-cycles spent to execute non-memory read scalar operations |
+| `SQ_THREAD_CYCLES_VALU` | Cycles  | Number of thread-cycles spent to execute VALU operations (this is similar to `INST_CYCLES_VALU`, but multiplied by the number of active threads) |
+| `SQ_WAIT_INST_LDS` | Qcycles | Number of quad-cycles spent waiting for LDS instruction to be issued.  |
 
 #### Local data share
 
-| Hardware Counter           | Unit   | Definition                                                |
-| :--------------------------| :------| --------------------------------------------------------: |
-| `sq_lds_atomic_return`       | Cycles | Number of atomic return cycles in LDS                     |
-| `sq_lds_bank_conflict`       | Cycles | Number of cycles LDS is stalled by bank conflicts         |
-| `sq_lds_addr_conflict[∗]`    | Cycles | Number of cycles LDS is stalled by address conflicts      |
-| `sq_lds_unaligned_stalls[∗]` | Cycles | Number of cycles LDS is stalled processing flat unaligned load/store ops |
-| `sq_lds_mem_violations[∗]`   | Count  | Number of threads that have a memory violation in the LDS |
+| Hardware Counter | Unit   | Definition |
+| :---------------------| :------| :------------------- |
+| `SQ_LDS_ATOMIC_RETURN` | Cycles | Number of atomic return cycles in LDS |
+| `SQ_LDS_BANK_CONFLICT` | Cycles | Number of cycles LDS is stalled by bank conflicts |
+| `SQ_LDS_ADDR_CONFLICT[∗]` | Cycles | Number of cycles LDS is stalled by address conflicts |
+| `SQ_LDS_UNALIGNED_STALL[∗]` | Cycles | Number of cycles LDS is stalled processing flat unaligned load/store ops |
+| `SQ_LDS_MEM_VIOLATIONS[∗]` | Count  | Number of threads that have a memory violation in the LDS |
+| `SQ_LDS_IDX_ACTIVE` | Cycles | Number of cycles LDS is used for indexed operations |
 
 #### Miscellaneous
```

Comment:
@SwRaw this is an empty heading; We should add content here or remove the following heading (Local data share).